backend/src/main/java/gov/cdc/usds/simplereport/db/repository/ApiUserRepository.java

```diff
@@ -30,4 +33,37 @@ public interface ApiUserRepository extends EternalSystemManagedEntityRepository<
 
   @Query(BASE_QUERY + " and loginEmail IN :emails" + NAME_ORDER)
   List<ApiUser> findAllByLoginEmailInOrderByName(Collection<String> emails);
+
+  String API_USER_ROLE_LEFT_JOIN = " LEFT JOIN api_user_role aur ON aur.apiUser = e";
+  String BY_ORG_AND_UNDELETED_USER = " WHERE aur.organization = :org AND e.isDeleted = false";
+  String JOINED_NAME_ORDER =
```

Comment:
Great question! Since I am using a `LEFT JOIN` I need to specify that I want to order by the `ApiUser`'s `nameInfo` and `internalId` since the one you mentioned `NAME_ORDER` doesn't make that distinction.

Let me know if there's a nicer way to write this!! 👀 Thank you!!

Reply:
I wonder if `NAME_ORDER` could be updated to specify the entity and we could just use that one everywhere instead of having two where one is just more explicit

Reply:
OOO good idea 😅 Will change that!

Reply:
Done! thank you!